src/tests/unit/tests/electron/platform/android/scan-controller.test.ts

```diff
@@ -208,92 +205,6 @@ describe('ScanController', () => {
         telemetryEventHandlerMock.verifyAll();
     });
 
-    it('scans and handle successful v2 response', async () => {
```

Comment:
Even though this test name includes "v2 response', it was actually processing a v1 response. The name was wrong.